From ca6fb4908bdb9f8be1ff67a5a8b0e94e0973d12f Mon Sep 17 00:00:00 2001
From: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
Date: Fri, 20 Nov 2020 22:47:42 +0100
Subject: [PATCH] remove images from SUBDIRS

The images/Makefile creates some test images so it is not crucial part.
Moreover, it produces some binaries using the cross-toolchain, tries
to run them during the same session and fails badly of course.

./mklarge ./minimal ./large
./mklarge: line 117: /build/.../hivex-1.3.18/images/.libs/lt-mklarge: No such file or directory

Signed-off-by: Maciej Pijanowski <maciej.pijanowski@3mdeb.com>
---
 Makefile.am  | 2 +-
 configure.ac | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e7e433819960..c3d0e674a328 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -20,7 +20,7 @@ ACLOCAL_AMFLAGS = -I m4
 # Work around broken libtool.
 export to_tool_file_cmd=func_convert_file_noop
 
-SUBDIRS = gnulib/lib generator lib images gnulib/tests
+SUBDIRS = gnulib/lib generator lib gnulib/tests
 
 if HAVE_LIBXML2
 SUBDIRS += xml
diff --git a/configure.ac b/configure.ac
index 28f2c054f9c4..5cbc22d09cd4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -556,7 +556,6 @@ AC_CONFIG_FILES([Makefile
                  gnulib/lib/Makefile
                  gnulib/tests/Makefile
                  hivex.pc
-                 images/Makefile
                  lib/Makefile
                  lib/tools/Makefile
                  ocaml/Makefile ocaml/META
-- 
2.25.1

